### PR TITLE
Navbar PRA-89, logout PRA-91, username in the session storage PRA-102

### PR DESCRIPTION
**Pull request by AnnaSolovykh**
## Requirements

1. PRA-89  Add navbar with search, add to favorites, logout buttons with appropriate routes
3. PRA-91
 1) The logout request is sent to backend successfully

2) Logout happens after clicking the logout text in Navbar

3) The user gets redirected to Home page

4) The username is stored in the session storage 

## Test Steps

1) Log in and see the navbar. 
2) Log out and check in the console that the logout has been successful. 
3) After the successful logout you are navigated to the Home page.
4) You can also check the mobile version with the hamburger menu.
5) You can see the user's name in the session storage when they are logged in.
## UI/UX Outcome

Screen Shot 2023-08-04 at 16 06 50

Screen Shot 2023-08-04 at 16 12 27

Screen Shot 2023-08-04 at 16 10 43

Screen Shot 2023-08-04 at 16 09 19

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaPestova1 — APPROVED** [review]
Everything works perfect and I like it! You did a great job. Could you please not comment your navbar (it is awesome) and make all routs leading on add recipe page, by now. Thanks.

**Review by AnnaPestova1 — APPROVED** [review]

**Review by AnnaPestova1 — APPROVED** [review]

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                  type="text"
+                  placeholder="Search a recipe..."
+                  id="search"
+                  variant="outline"
+                /> 
+              <HStack as={"nav"} spacing={20} display={{ base: "none", md: "flex" }} justifyContent={"flex-end"}>
+                <NavLink to="/search-choice">ADD RECIPE</NavLink>
+                <NavLink to="/app" >SAVED</NavLink>
+                <NavLink to="/app">PLANNER</NavLink>
+                <NavLink to="/app">SHOPPING</NavLink>
```
Comment (Ekaterina-Bondareva) [inline src/components/Layout.tsx:73]: According to requirements: "PRA-89 Add navbar with search, add to favorites, logout buttons with appropriate routes". But I do not see appropriate routes.

**Review by Ekaterina-Bondareva — CHANGES_REQUESTED** [review]
Good job! Please find some comments.

File: src/components/Layout.tsx (hunk 1 above)
Comment (AnnaSolovykh) [reply src/components/Layout.tsx:73]: Kate, appropriate routes do not exist at this point, they will be created as stretch goals. Team wants to see the whole navbar. 
Please approve PR, your comment is not related to the issue. I write the ticket to myself and easily can change the wordings if this is the issue.

Comment (AnnaSolovykh) [reply src/components/Layout.tsx:73]: Design has also changed, and add to favorites is not in the navbar anymore. In this pr there are only two links with real routes, and they have them.

**Review by Ekaterina-Bondareva — APPROVED** [review]
Great job!

**Pull request merged**